bindings/java/src/test/java/org/apache/opendal/AsyncStepsTest.java

```diff
@@ -40,7 +40,7 @@ public void a_new_open_dal_async_operator() {
 
     @When("Async write path {string} with content {string}")
     public void async_write_path_test_with_content_hello_world(String fileName, String content) {
```

Comment:
Can we capture a bytes directly?

Reply:
From my quick look, Cucumber's built-in type does not support byte array.

Reference: https://github.com/cucumber/cucumber-expressions#parameter-types

Reply:
Thanks!